src/@types/settings.ts

```diff
@@ -142,14 +142,15 @@ export interface Payments {
   feeSchedules: FeeSchedules
 }
 
-export interface ZebedeePaymentsProcessor {
+export interface PaymentsProcessor {
```

Comment:
Please add a new type equal  to ZebedeePaymentsProcessor or just add new interface called LnbitsProcessor.